tool/travis.sh

```diff
@@ -57,5 +57,5 @@ else
   (cd testing/test_package_small; dart -c ../../bin/dartdoc.dart)
 
   # Run the tests.
-  pub run grinder test
+  pub run test
```

Comment:
Why the change?

Reply:
See comments on this commit: https://github.com/dart-lang/dartdoc/pull/1489/commits/90a379d9d5845616cf8b8a050c15bc4fda5e4c89